### PR TITLE
Remove unused variable: compressionFlag

### PR DESCRIPTION
**Pull request by isimluk**
Unused since 1f7a56c869614f632d22e750857293de5153f7b2

Signed-off-by: Šimon Lukašík <isimluk@fedoraproject.org>

### CODE REVIEW

**Comment by rhatdan** [comment]
Looks like you created a gofmt error?

File: layers.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ import (
 const (
 	tarSplitSuffix  = ".tar-split.gz"
```
Comment (nalind) [inline layers.go]:
```suggestion
	tarSplitSuffix = ".tar-split.gz"
```

File: layers.go [hunk 2]
```diff
@@ -30,7 +30,6 @@ import (
...
 	incompleteFlag  = "incomplete"
```
Comment (nalind) [inline layers.go]:
```suggestion
	incompleteFlag = "incomplete"
```

**Comment by rhatdan** [comment]
LGTM
@nalind @mtrmac @TomSweeneyRedHat @giuseppe PTAL

**Comment by mtrmac** [comment]
Sure.

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**